src/com/palmergames/bukkit/towny/listeners/TownyPlayerListener.java

```diff
@@ -1003,6 +1003,77 @@ public void onJailedPlayerUsesCommand(PlayerCommandPreprocessEvent event) {
 		}
 	}
 	
+	/** 
+	 * Allows restricting commands while being in a town.
+	 * Also allows limiting commands to personally-owned plots only.
+	 * Works almost the same way as jail command blacklisting, except has more stuff
+	 * @param event PlayerCommandPreprocessEvent
+	 */
+	@EventHandler(priority = EventPriority.NORMAL)
+	public void onPlayerUsesCommandInsideTown(PlayerCommandPreprocessEvent event) throws NotRegisteredException {
```

Comment:
No NRE is thrown anywhere, so this can be removed.
```suggestion
	public void onPlayerUsesCommandInsideTown(PlayerCommandPreprocessEvent event) {
```